src/Components/Continents.jsx

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable no-unused-vars */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable no-undef */
+/* eslint-disable no-tabs */
+/* eslint-disable no-mixed-spaces-and-tabs */
+/* eslint-disable max-len */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Link,
+  Routes,
+  Route,
+  useNavigate,
+  useLocation,
+} from 'react-router-dom';
+import { MdAdsClick } from 'react-icons/md';
+import { IoArrowBack } from 'react-icons/io5';
+import Countries from './Countries';
+import { fetchCountries } from '../myredux/myreducer';
+
+const Continents = (props) => {
+  const navigate = useNavigate();
+  const dispatch = useDispatch();
```

Comment:
- [ ] You failed to add tests. Kindly add tests using React testing library and jest.